contracts/Aavegotchi/libraries/LibAppStorage.sol

```diff
@@ -283,7 +283,7 @@ struct AppStorage {
     string symbol;
     //Addresses
     address[] collateralTypes;
-    address ghstContract;
+    // address ghstContract;
```

Comment:
@cinnabarhorse are we planning to upgrade the current diamond? if yes, this will cause a storage layout issue with the current diamond, better to leave the ghstContract variable there although unused
```suggestion
     address ghstContract;
```

Reply:
No, we're not deploying this change to the legacy diamond